docs/contributor_guide/developer_call_notes/2020_10_08.rst

```diff
@@ -0,0 +1,46 @@
+Developer Call 2020-10-08
+-------------------------
+
+Scribbler: `Richard Edgar <https://github.com/riedgar-ms>`_
+
+Attendees:
+
+- `Miro Dudik <https://github.com/MiroDudik>`_
+- `Richard Edgar <https://github.com/riedgar-ms>`_
+- `Adrin Jalali <https://github.com/adrinjalali>`_
+- `Diondra Peck <https://github.com/diondrapeck>`_
+- Alex Quach
+- `Kevin Robinson <https://github.com/kevinrobinson>`_
+- `Mehrnoosh Sameki <https://github.com/mesameki>`_
+- `Hanna Wallach <https://www.microsoft.com/en-us/research/people/wallach/>`_
+- `Hilde Weerts <https://github.com/hildeweerts>`_
+
+Agenda:
+
+- Requirements for next release
+    - Still need new metrics API
+    - Need to announce deprecation of ``FairnessDashboard``
+    - Need to make sure Quickstart will actually work
+      (short term fix: put note about running `pip install` against GitHub at the top)
+    - Not clear what ``ThresholdOptimizer`` doc fixes Roman had in mind
+      but they probably shouldn't block the release
+
+- Governance
+
+    An update from Miro. Probably won't be ready for the October NumFocus application
+    deadline, but Microsoft will hopefully separate Fairlearn as a neutral organisation
+    during October. This will remove the need for things like `Security.md` which are not
+    particularly open source friendly.
+
+- Naming the metrics API
+
+    With the basic API for the new style metrics agreed, we need to decide on names for
+    everything. This is currently
+    `an open issue <https://github.com/fairlearn/fairlearn-proposals/issues/17>`_
+    and *really* needs to be brought to resolution.
+
+    We had a lengthy discussion. There was a feeling that `MetricFrame` would be better than
+    `GroupedMetric` for the whole class, but we did not agree on alternatives for
+    `conditional_features=`. At Kevin's suggestion, RGE will rewrite the example code
+    introducing the new metric API to be focussed on the sociotechnical side, rather than
```

Comment:
@riedgar-ms You know I'm super in favor of having all examples in the project follow the [contributor guidelines](https://fairlearn.github.io/contributor_guide/contributing_example_notebooks.html) :)  but fwiw during the meeting it «was @MiroDudik who brought this up.

My suggestion was really about "when thinking about how to name an API, it can help to think about it in terms of how users will first come to see and experience the API."  Concretely, that means even for the purely technical work of naming and API design, a helpful way to get feedback is to focus on the quickstart example that new users will first encounter, or the the blog post announcing the new release.  That also lets you quickly and informally test how clear it is with some folks before releasing too.